decorator/timer.py

```diff
@@ -0,0 +1,23 @@
+import time
```

Comment:
Inna sprawa natury stylistycznej (znowu na bazie [PEP-8](https://peps.python.org/pep-0008/)):
- odstęp między funkcjami a resztą kodu lub klasami a resztą kodu to powinny być dwie puste linijki
- na końcu pliku powinna być jedna pusta linijka